src/legacy/core_plugins/kibana/public/discover/get_inner_angular.ts

```diff
@@ -88,6 +88,7 @@ import { createRenderCompleteDirective } from './np_ready/angular/directives/ren
  * needs to render, so in the end the current 'kibana' angular module is no longer necessary
  */
 export function getInnerAngularModule(name: string, core: CoreStart, deps: DiscoverStartPlugins) {
+  initAngularBootstrap();
```

Comment:
This is the new way how to load angular-bootstrap modules. They still have to be listed as dependencies in the angular app modules.